R/sparklyr.sedona/DESCRIPTION

```diff
@@ -0,0 +1,29 @@
+Type: Package
+Package: sparklyr.sedona
+Title: Sparklyr Extension for Apache Sedona
+Version: 0.1.0
+Authors@R:
```

Comment:
Could you change the maintainer name to Apache Sedona? See here: https://github.com/apache/incubator-sedona/blob/master/python/setup.py#L33

Reply:
Sure. There is one logistical matter we need to think about though: when submitting this R package to CRAN, there will be an email from CRAN to the maintainer's email address containing a confirmation link, and someone will need to click that link each time. Would dev@sedona.apache.org work for that purpose?

Reply:
If it's a mailing list then that should be fine. I imagine we will vote before starting a release of this R package to CRAN anyways.